spec/robots/dor_repo/gis_assembly/extract_boundingbox_spec.rb

```diff
@@ -378,5 +378,6 @@
   it 'updates the cocina with the bounding box' do
     test_perform(robot, druid)
     expect(object_client).to have_received(:update) { |args| expect(args[:params].description.to_h).to match Cocina::Models::Description.new(expected_description).to_h }
+    expect(IO).to have_received(:popen).with("gdalinfo 'MCE_AF2G_2010.tif'")
```

Comment:
Shouldn't this be getting called with the directory in the path?

Reply:
I didn't think so, but the integration test proved me wrong. Fixing.